api/paidAction/transfer.js

```diff
@@ -0,0 +1,57 @@
+import { satsToMsats } from '@/lib/format'
+import { notifyDeposit } from '@/lib/webPush'
+import { PAID_ACTION_PAYMENT_METHODS } from '@/lib/constants'
+
+export const anonable = true
+
+export const paymentMethods = [
+  PAID_ACTION_PAYMENT_METHODS.P2P
+]
+
+export async function getCost ({ sats }) {
+  return satsToMsats(sats)
+}
+
+export async function invoiceablePeer ({ targetUserId }, { models }) {
+  return targetUserId
+}
+
+export async function perform ({ invoiceId, sats, description, descriptionHash, comment, targetUserId }, { me, tx }) {
+  const invoice = await tx.invoice.update({
+    where: { id: invoiceId },
+    data: {
+      comment
+    }
+  })
+  await notifyDeposit(targetUserId, invoice)
+  return { sats, targetUserId }
+}
+
+export async function describe ({ sats, description, descriptionHash }, context) {
+  return `SN: ${description ?? ''}`
+}
+
+export async function describeHash ({ sats, description, descriptionHash }, context) {
+  return descriptionHash
+}
+
+export async function getSybilFeePercent ({ sats, description, descriptionHash }, context) {
+  return 10n
+}
+
+export async function onPaid ({ invoice }, { tx }) {
+  const isP2P = !!invoice.invoiceForward
+  if (isP2P) return
+  if (!invoice?.msatsReceived) throw new Error('Not paid?')
```

Comment:
Did something change where somethings calling `onPaid` isn't doing this check?

Reply:
nope, you are right this check is useless